### ISSUE TITLE
static color pixel count by port/string

### ISSUE THREAD

**Issue by bloojoop** [FalconChristmas/fpp#1793]
Can we change the pixel count test modes to be static color instead of cycle color.  It can be slightly difficult to count on some models/props when the color bands continually change.  

Maybe change the two pixel count test modes or make a choice for cycle/static color bands.  I'm not sure the choice would be needed but maybe an option?

**Comment by darylc** [comment]
@dkulp Can you indicate would implement this at some point or if we should close it?

**Comment by darylc** [comment]
@computergeek1507 how did you go?

**Comment by juddge** [comment]
I agree with this one, the flashing colours is annoying, i would like really like this as a option on the testing
